pkgs/top-level/ocaml-packages.nix

```diff
@@ -326,6 +326,8 @@ let
 
     js_of_ocaml-compiler = callPackage ../development/tools/ocaml/js_of_ocaml/compiler.nix {};
 
+    js_of_ocaml-lwt = callPackage ../development/tools/ocaml/js_of_ocaml/lwt.nix {};
```

Comment:
Probably at some point it would be worth to introduce a `js_of_ocaml` namespace and add abstraction to wrap packages below that automatically.

Reply:
How many would be such a “point”? I have one more such jsoo library to add.

Reply:
How many packages are applicable? Usually it is better to refactor early because the amount of refactoring increases with the number of packages.

Reply:
It could look like altcoins: https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/altcoins/default.nix